docs/UserGuide.md

```diff
@@ -216,9 +216,20 @@ Adds an assignment with a deadline to all students in the specified module .
 
 Format: `giveall m/MODULE d/DESCRIPTION by/ d/M/YYYY [,HHmm]`
 
+**Note:**
+1. It is possible to add an assignment using `giveall` even if some students have the assignment already. Students who have the 
+assignment will not receive a duplicate assignment.
+2. When using `giveall` in the situation stated in point 1, the specified assignment in the input needs to have the same 
+due date as the existing assignment. **The command will not be valid if the due date is different.** 
+Point 3 of the examples section illustrates this.
```

Comment:
There is no numbered points in the user guide for examples. Possible to omit the "point x" and say `If some students have the same assignments you wish to give but of a different due date, the command will not work to ensure consistency."?

If you take my suggestion, maybe can put this point in a note box